src/config.js

```diff
@@ -0,0 +1,10 @@
+//const env = process.env.NODE_ENV; // 'dev' or 'test'
+
+const config = {
+ host: `localhost`,
+ port: 8081,
+ AppID: `7c0c8e717c744b103b263ef8b89d17cc429e6cfab0cddb8b3df21b66e21b45c1`,
+ APP_SECRET: `1bb0b4cf625557ee3260a756dc63bbc8544247dacd0685513fd9cf87e7f66afb`,
```

Comment:
Don't EVER commit any secret and push to remote. Please remove them.

You should add an example file, put the file that contains real credentials in git ignore. And you should load the value from environment variable instead.